orbitx/physics/calc.py

```diff
@@ -49,7 +49,7 @@ def orb_speed(habitat: Entity,
     """The orbital speed of an astronomical body or object.
     Equation referenced from https://en.wikipedia.org/wiki/Orbital_speed"""
     return math.sqrt(
-        (reference.mass**2 * common.G) /
+        (reference.mass ** 2 * common.G) /
```

Comment:
Can you keep this formatted with autopep8? idk if pycharm has a convenient plugin for it.

If there's an easy way to have autopep8 automatically format, that'll keep the code style consistent and formatting changes like this away.

But it's just a small thing so if it's not easy, don't bother.

Reply:
Same for all the other lines in this file

Reply:
Weird! Pycharm kept giving me formatting suggestions that I accepted. I must have accepted a formatting change for the whole file o.0. I'll look into it.